CHANGELOG.md

```diff
@@ -74,6 +74,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#5741](https://github.com/thanos-io/thanos/pull/5741) Query: add metrics on how much data is being selected by downstream Store APIs.
 - [#5673](https://github.com/thanos-io/thanos/pull/5673) Receive: Reload tenant limit configuration on file change.
 - [#5749](https://github.com/thanos-io/thanos/pull/5749) Query Frontend: Added small LRU cache to cache query analysis results.
+- [#5773](https://github.com/thanos-io/thanos/pull/5773) Store: Support disable cache index header file.
```

Comment:
Let's move the changelog entry to the unreleased section.

Reply:
Done